cmd/activator/main.go

```diff
@@ -285,7 +285,7 @@ func main() {
 		name, server := "https", pkgnet.NewServer(":"+strconv.Itoa(networking.BackendHTTPSPort), ah)
 		go func(name string, s *http.Server) {
 			s.TLSConfig = &tls.Config{
-				MinVersion:     tls.VersionTLS12,
+				MinVersion:     tls.VersionTLS13,
```

Comment:
It seems TLS 1.3 between `Ingress -> activator` has an issue - probably related to https://github.com/envoyproxy/envoy/issues/9300

Could you try to revert this line if it will make CI pass? I think we should change this TLS min version carefully or at least   by separate PR.

Reply:
Testing this suggestion: tests running.